usr/share/rear/finalize/Fedora/i386/550_rebuild_initramfs.sh

```diff
@@ -88,7 +88,7 @@ for INITRD_IMG in $( ls $TARGET_FS_ROOT/boot/initramfs-*.img $TARGET_FS_ROOT/boo
     # and then "rear recover" cannot be aborted with the usual [Ctrl]+[C] keys.
     # Use plain $var because when var contains only blanks test "$var" results true because test " " results true:
     if test $mkinitrd_binary ; then
-        if chroot $TARGET_FS_ROOT $mkinitrd_binary -v -f ${WITH_INITRD_MODULES[@]} $INITRD $kernel_version >&2 ; then
```

Comment:
I believe that here it might be best to quote the original form:
"${WITH_INITRD_MODULES[@]}"
Since modules (hopefully) do not have whitespace in them, it does not matter, but the code could be copied somewhere else.

Reply:
Actually, `WITH_INITRD_MODULES` is currently not an array (similarly to `INITRD_MODULES`), so using an array expansion does not even make sense.

Reply:
https://github.com/rear/rear/pull/2771/commits/2fbb8ed8230e4af972d3a5933c6505c852950866
handles INITRD_MODULES and WITH_INITRD_MODULES as strings